src/api-engine/api/routes/channel/views.py

```diff
@@ -177,32 +203,116 @@ def update(self, request, pk=None):
         """
         serializer = ChannelUpdateSerializer(data=request.data)
         if serializer.is_valid(raise_exception=True):
-            peers = serializer.validated_data.get("peers")
             channel = Channel.objects.get(id=pk)
+            org = request.user.organization
             try:
-                org = request.user.organization
-                org_name = org.name
-                msp_id = org_name.split(".")[0].capitalize()
-                dir_node = "{}/{}/crypto-config/peerOrganizations".format(
-                    CELLO_HOME, org_name)
-                peer_cli_envs = {
-                    "CORE_PEER_LOCALMSPID": msp_id,
-                    "CORE_PEER_MSPCONFIGPATH": "{}/{}/users/Admin@{}/msp".format(
-                        dir_node, org_name, org_name),
+                data = request.FILES["data"]
+                config = data.config
+                org = data.org
```

Comment:
There should be an optional param to specify whether it's Application org (by default) or Orderer org.